analytics.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import re
+import time
+from datetime import datetime
+
+
+
+DELAY = 0.1
+
+
+def get_trees():
+    r = requests.get('https://teamtrees.org')
+    return int(re.search('data-count="(\\d+)"', r.text).group(1))
+
+
+def main():    
+        now = datetime.now()
+        last_trees = get_trees()
+        #print(last_trees)
+        totalLoops = 0
+        loops = 0
+        gained_total = 0
+        while True:
+            trees_now = get_trees()
```

Comment:
There should be a delay between requests, without it(when there are no donations) it might be DDOSing the website